README.md

```diff
@@ -120,31 +121,32 @@ as a Kubernetes master for other nodes!
    securely, since leaking it can compromise your cluster.
 
 2. On the worker node you want to join to the cluster, install the
-   pre-requisites. This is the same script
-   used for setting up the master too.
+   pre-requisites:
    ```bash
+   git clone https://github.com/data-8/kubeadm-bootstrap
+   cd kubeadm-bootstrap
    sudo ./install-kubeadm.bash
    ```
    
-   This installs kubeadm, a supported version of docker and sets up the
+   This installs `kubeadm`, a supported version of docker and sets up the
    appropriate storage driver options for docker.
 
-4. Setup the node! Copy the `kubeadm join` command you got as output
-   of step (1) from the maser, and run it in the node. You might have to
-   prefix it with `sudo`. This should take a few minutes.
+3. Copy the `kubeadm join` command you got as output
+   of step (1) from the master, prefix with `sudo` and run it. 
+   This should take a few minutes.
    
-5. Test that everything is up!
+4. Test that everything is up!
 
-   a. On the master, run `kubectl get node` - it should list your new node in
+   a. On the master, run `kubectl get node`.  It should list your new node in
       `Ready` state.
 
-   b. Run `kubectl --namespace=kube-system get pod -o wide`. This should show
-      you a `kube-proxy`, a `flannel` and `nginx-controller` pod running on your
+   b. On the master, run `sudo kubectl --namespace=kube-system get pod -o wide`. This should show
```

Comment:
kube-controller should only be running on the master node, not on every node.

Reply:
I just wanted to make that explicit for newbies (like me).

Reply:
But would you prefer me to remove it?

Reply:
ping @yuvipanda